lldb/include/lldb/Core/Progress.h

```diff
@@ -55,6 +55,11 @@ namespace lldb_private {
 
 class Progress {
 public:
+  /// Enum that indicates the type of progress report
+  enum class ProgressReportType {
+    eAggregateProgressReport,
+    eNonAggregateProgressReport
+  };
```

Comment:
This might need to belong in lldb-enumerations if we want `SBDebugger::GetProgressFromEvent(...)` to have an overload where this is supplied.